SConstruct

```diff
@@ -2195,6 +2191,8 @@ if env["WITH_GL"] and doConfigure :
 				os.path.basename( imageEnv.subst( "$INSTALL_LIB_NAME" ) ),
 				os.path.basename( sceneEnv.subst( "$INSTALL_LIB_NAME" ) ),
 				"OpenImageIO$OIIO_LIB_SUFFIX",
+				"GLEW$GLEW_LIB_SUFFIX",
+				"boost_wave$BOOST_LIB_SUFFIX",
```

Comment:
As above, I think this means that the check for `boost_wave` can fail because of a missing `GLEW`, and vice versa.